.githooks/pre-commit

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#set -x
+PREFIX="pre-commit:"
+RESULTS_FILE=tfsec-scan-output.txt
```

Comment:
Quote your variables please.

Also, it would help if there were some useful comments - reading comments aids understanding.

Reply:
ok